stm32-data-gen/src/chips.rs

```diff
@@ -284,7 +284,9 @@ impl PeriMatcher {
             ("STM32H50.*:EXTI:.*", ("exti", "h50", "EXTI")),
             ("STM32H5.*:EXTI:.*", ("exti", "h5", "EXTI")),
             (".*:EXTI:.*", ("exti", "v1", "EXTI")),
-            ("STM32L0.*:CRS:.*", ("crs", "l0", "CRS")),
+            ("STM32L0.*:CRS:.*", ("crs", "v1", "CRS")),
+            ("STM32G0.*:CRS:.*", ("crs", "v1", "CRS")),
```

Comment:
I'm not quite sure if all G0 chips have the CRS peripheral. Is there a better way to constrain this down or does it matter?